examples/simple_window/src/main.rs

```diff
@@ -0,0 +1,59 @@
+#![windows_subsystem = "windows"]
+
+use std::convert::TryFrom;
+
+use project_reunion_rs::{PackageVersion, workarounds};
+
+use bindings::{
+    Microsoft,
+    Microsoft::UI::Xaml::{
+        Application, ApplicationInitializationCallback, Controls::Button, HorizontalAlignment,
+        LaunchActivatedEventArgs, RoutedEventHandler, Window,
+    },
+};
+
+use windows::{implement, IInspectable, Interface, HRESULT};
+
+#[implement(extend Microsoft::UI::Xaml::Application, override OnLaunched)]
+struct App {
+    window: Option<Window>,
+}
+
+#[allow(non_snake_case)]
+impl App {
+    fn OnLaunched(&mut self, _: &Option<LaunchActivatedEventArgs>) -> windows::Result<()> {
+        self.window = Some(Window::new()?);
+        if let Some(window) = &self.window {
+            window.SetTitle("WinUI Desktop, Unpackaged (Rust)");
+
+            let button = Button::new()?;
+            button.SetContent(IInspectable::try_from("Click Me")?);
+            button.SetHorizontalAlignment(HorizontalAlignment::Center);
+            button.Click(RoutedEventHandler::new(|sender, _args| {
+                if let Some(button) = sender {
```

Comment:
I'm still hoping to simplify this syntax - but Rust really isn't making it easy. 😉